### DISCUSSION TITLE
How to collapse notification by default or make more compact?

### DISCUSSION THREAD

**Discussion by Jaluvshuskies** [breezy-weather/breezy-weather#1248] [Q&A]
Hello,

Basically, this question is for the notification widget. Is there any way to either/both:
A. Auto collapse the notification widget
B. Make it smaller/shorter or more compact somehow?

This might end up being more of a suggestion than help for existing features - I looked in settings but didn't see a way to accomplish either of these. It takes up a lot of notification drawer space, so I would love a way to make it more compact without doing it manually every time with the ^ (it goes back to expanded)

I am using the hourly forecast style, which I like, and did put it as silent
I am on A14/OOS 14 on the OnePlus Open

**Comment by papjul** [comment]
Not exactly what you asked but in Widget settings, tap "Style", choose "Native". Let me know if it can fit your need ;)

**Reply by Jaluvshuskies** [reply]
> Not exactly what you asked but in Widget settings, tap "Style", choose "Native". Let me know if it can fit your need ;)

That helps!! If there's no other way to auto collapse or make specifically the weekly view compact, I'll just do this

**Reply by papjul** [reply]
If you don't look at the daily view that often but you need it, an homescreen widget of the daily view can be an option
